_posts/guides/2021-12-21-guide_gamemode-info-surf.md

```diff
@@ -27,4 +27,7 @@ Surf is a very popular gamemode
 
 ## How Surf Works
 
-TODO
\ No newline at end of file
+You can surf by holding A on the right side of the ramp and D on the left without holding W.
+When getting on a ramp you need to look downwards, then slowly turn your camera until the bottom where you should be looking flat with the ramp.
+If you want to jump you need to slowly start looking at the ramp and when you're halfway up look directly at it.
+When you are mid-air its beneficial to strafe by looking left while holding A then switcing to looking right while holding D and repeating that.
```

Comment:
Typo on "switching". I don't like the "and repeating that", I think you're talking about strafing for speedgame somewhat but that's not necessary, the point should be about always holding the correct strafe key when turning.

Reply:
Still not too happy with this, still the issue in the latter part of my above comment.